templates/partials/summary/summary.html

```diff
@@ -10,6 +10,7 @@
         "groups": [
           {
             "groupTitle": '<div id="' + group.id +'">' + group.title + '</div>' if group.title else None,
+            "headers": ["Question", "Answer Given", "Change Answer"],
```

Comment:
The capitalisation doesn't match the card:
```suggestion
            "headers": ["Question", "Answer given", "Change answer"],
```